app/controllers/admin/coupons_controller.rb

```diff
@@ -1,21 +1,23 @@
 module Admin
   class CouponsController < Admin::BaseController
-    include Orderable
     before_action :set_coupons, only: %i[show edit destroy update]
-    helper_method :sort_column, :sort_direction
 
     def index
-      #@users = User.all
       @pagy, @coupons = pagy(Coupon.all, items: 5)
       respond_to do |format|
         format.html
         format.csv { send_data Coupon.all.to_csv, filename: "coupons-#{Date.today}.csv" }
       end
+
       if params[:search].present?
-       @pagy, @coupons =pagy( Coupon.search(params[:search]), items: 5)
-       @pagy, @coupons = pagy(@coupons.order(sort_column(@coupons) + ' ' + sort_direction), items: 5) if sort_column(@coupons).present? && sort_direction.present?   
+        @pagy, @coupons = pagy(Coupon.search(params[:search]), items: 5)
```

Comment:
Also rather than applying `params[:search]` presence check here. Can we always call the search method and early return is search params is blank? In that way, we can avoid conditional altogether?